app/assets/javascripts/initializers/initializeAdditionalContentBoxes.js

```diff
@@ -32,6 +32,7 @@ function initializeAdditionalContentBoxes() {
             el.innerHTML = html;
             initializeReadingListIcons();
             initializeAllFollowButts();
+            initializeUserFollowButts();
```

Comment:
You can replace `'use strict';` at the top of the file with `/* global initializeUserFollowButts */`. The `'use strict';` is not required and the global comment indicates to eslint that `initializeUserFollowButts` is already defined somewhere else globally. This should get rid of one of the code climate issues.